tool/site/client-react/src/components/prescription-tracker/prescriptionDetails.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from "react"
+import {SummaryList} from "nhsuk-react-components"
+import {PrescriptionProps} from "./prescription"
+import {Task} from "fhir/r4"
+import {formatNhsNumber} from "../../formatters/demographics"
+import {formatDate} from "../../formatters/dates"
+import {getCourseOfTherapyTypeExtension} from "../../fhir/customExtensions"
+
+export interface PrescriptionDetailProps {
+  id: string
+  type: string
+  patientNhsNumber: string
+  creationDate: string
+  status: string
+}
+
+export function createPrescriptionDetailProps(task: Task): PrescriptionDetailProps {
+  const prescriptionType = getCourseOfTherapyTypeExtension()(task.extension).valueCoding.code
+  return {
+    id: task.focus.identifier.value,
+    type: prescriptionType.substring(0, 1).toUpperCase() + prescriptionType.substring(1),
+    patientNhsNumber: formatNhsNumber(task.for.identifier.value),
+    creationDate: formatDate(task.authoredOn),
+    status: task.businessStatus.coding[0].display
```

Comment:
could we create a valueSet to match a code to display? https://simplifier.net/packages/uk.nhsdigital.medicines.r4/2.1.14-alpha/files/374928

Reply:
we could, but id rather we thought that through a little in a separate pr, i think some of our apis are a bit too open

Reply:
maybe wait till we get some user feedback

Reply:
sure